src/input.hpp

```diff
@@ -68,7 +68,8 @@ struct KeyboardInput : public Input, public EventListener {
             OnEventCallback on_event_callback = OnEventCallback{}
     )
         : Input{ target_tetrion, std::move(on_event_callback) },
-          m_controls{ controls } { }
+          m_controls{ controls },
+          m_event_buffer{ std::vector<SDL_Event>{} } { }
```

Comment:
Why should this be an error? An `std::vector` has a default construtor and thus `m_event_buffer` should be initialized automatically 🤔